tests/acceptance/pageObjects/favoritesPage.js

```diff
@@ -13,6 +13,20 @@ module.exports = {
       return this.navigate(this.url()).waitForElementPresent(
         this.page.FilesPageElement.filesList().elements.anyAfterLoading
       )
+    },
+    getBatchActionButtonPresentStatus: async function () {
+      let presentStatus = null
+      await this.isVisible('@batchActionButton', 'suppressNotFoundErrors', (res) => {
+        presentStatus = res.status
+      })
+      return presentStatus
+    }
+  },
+  elements: {
+    batchActionButton: {
+      selector:
+        '//button[@id="download-selected-btn"]//ancestor::div[@class="uk-grid-small uk-grid"]',
```

Comment:
I don't think it's reliable to use `uk-toolkit` selectors. can we use more reliable selector here?